docs/modules/ROOT/pages/getting-started/relationships-and-advanced-filtering.adoc

```diff
@@ -1,41 +1,46 @@
-= Relationships and complex filtering
+= Relationships and advanced filtering
 
-This tutorial shows how to add relationships and more complex filters, and alias the results of the projection of the Cypher query from the xref:getting-started/filters-and-projections.adoc[previous tutorial].
+This tutorial shows how to add relationships and more advanced filters, as well as alias the results of the projection.
+It uses the Cypher query example from the xref:getting-started/filters-and-projections.adoc[Filters and projections] tutorial and the link:https://neo4j.com/docs/getting-started/appendix/example-data/#built-in-examples[Movies Dataset].
 
-The final query will look like this:
 ```cypher
 MATCH(m:Movie)<-[r:ACTED_IN]-(a:Person)
 WHERE (a.name="Keanu Reeves" AND (NOT(m.title CONTAINS "Matrix") OR m.released < 2000))
 RETURN m.title, m.tagline, m.released, r.roles AS actingRoles
 ```
 
-This query matches a Movie along with its actors, filtering movies with the actor `Keanu Reeves`. It also filters on either movies **not** containing `Matrix` or released before 2000. Finally, the projection also includes the property `roles` from the relationship, aliased to the name `actingRoles`.
+This query matches a `Movie` along with its actors. 
+It should only return movies with the actor `Keanu Reeves`, excluding any movie with a title containing `Matrix` released after 1999.
+It also adds the property `roles` from the relationship to the projection, aliasing it to `actingRoles`.
 
-If you try this query on the Movies dataset, you'll get the following results:
+Querying it on the Movies Dataset should prompt this result:
 
 ```
-╒════════════════════════╤════════════════════════════════════════════════════════╤══════════╤═══════════════════╕
+╒════════════════════════╤════════════════════════════════════════════════════════╤══════════╤
```

Comment:
I'm not sure what happened here with the formating but the table doesn't look like a table anymore

If formatting is a problem, maybe we could use an actual table here? :thinking: